examples/light-switch-app/realtek_bee/README.md

```diff
@@ -0,0 +1,15 @@
+---
+orphan: true
+---
+
+# Matter BEE4 SDK
+
+## Realtek SDK
+
+More detailed information on the Bee4 SDK can be found in the
+[Realtek Matter SDK](https://github.com/rtkconnectivity/rtl87x2g_sdk).
```

Comment:
There should be some instructions on how to compile in the SDK itself. likely some build-examples.py line.